app/main.py

```diff
@@ -1,13 +1,32 @@
+from __future__ import annotations
+
+from typing import Dict, List
+
+
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error.get("line_number"),
```

Comment:
Here use the same syntax like in other values of this dict

Reply:
yes, please, use only one style: `get()` or `[]` everywhere